src/base/rss/rss_feed.cpp

```diff
@@ -207,13 +207,14 @@ void Feed::handleParsingFinished(const RSS::Private::ParsingResult &result)
         m_lastBuildDate = result.lastBuildDate;
 
         foreach (const QVariantHash &varHash, result.articles) {
-            auto article = Article::fromVariantHash(this, varHash);
-            if (article) {
+            try {
+                auto article = new Article(this, varHash);
                 if (!addArticle(article))
                     delete article;
                 else
                     m_dirty = true;
             }
+            catch (...) {}
```

Comment:
@glassez:  I too find myself wondering what this particular line is supposed to do.  The `catch {}` is one thing, but why the `(...)`?